learn/cloud/analytics.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: Analytics
+description: Meilisearch Cloud offers in-depth search analytics to help you understand how users search in your application.
+sidebarDepth: 3
+---
+
+# Analytics
+
+Enable Meilisearch Cloud analytics to gain insight on what your users are searching for, how often they find it, and how much they have to wait between typing a query and receiving results.
+
+## Requirements
+
+You must have a Meilisearch Cloud account to have access to search analytics.
+
+## Activating analytics
+
+### Enable analytics in the project overview
+
+Log into your Meilisearch Cloud account and navigate to your project's overview. Find the "Analytics" section and click on the "Enable analytics" button:
```

Comment:
I generally use italics when quoting the UI copy, and I thought that was the practice. Just happy to know more about this / agree on something so we can edit the editing guidelines